shared/common-adapters/floating-menu/index.tsx

```diff
@@ -38,6 +47,7 @@ export default (props: Props) => {
       attachTo={props.attachTo}
       style={props.containerStyle}
       propagateOutsideClicks={props.propagateOutsideClicks}
+      key={itemChangeCounter.toString()}
```

Comment:
No idea what's going on here, but without this forced re-render it just shows stale data.

Reply:
Did you merge master for my fix and run modules.  Let's not do this

Reply:
I think so; this is pretty new-ish. OK I’ll try to figure out this weekend